src/tables.jl

```diff
@@ -86,7 +84,9 @@ function Tables.schema(cs::Columns)
     return Tables.Schema(map(Symbol, column_names(jl_result)), types)
 end
 
-function Column(jl_result::Result, col::Integer, name=Symbol(column_name(jl_result, col)))
+function Column(
+    jl_result::Tresult, col::Integer, name=Symbol(column_name(jl_result, col))
+) where {Tresult<:Result}
```

Comment:
Does this particular specialization have an impact? If so, perhaps we had caused a performance issue when we introduced binary format as a type parameter.

Reply:
I believe so -- the original changes had some redundancy, but I'll double check on Monday

Reply:
Can confirm, the specialisation on the type of `Result` is important. Removing it significantly hurts performance

Reply:
@samuel-massinon That's an interesting quirk we may have missed.